@here/harp-text-canvas/lib/TextCanvas.ts

```diff
@@ -417,9 +425,16 @@ export class TextCanvas {
         if (clear === true) {
             this.m_renderer.clear(true);
         }
-        for (const layer of this.m_layers) {
-            layer.storage.update();
-            this.m_renderer.render(layer.storage.scene, camera);
+        for (let i = 0; i < this.m_layers.length; i++) {
+            const layer = this.m_layers[i];
+            if (layer.id >= (lowerLayerId ?? 0)) {
```

Comment:
Just a note, I guess we don't have to chage it in this PR:
this.m_layers is sorted by id, see addLayer. Even if it's not important for these small arrays, we should be to easily do binary search, lower/upper bounds in sorted arrays. For some funny reason javascript doesn't have that (essential) functionality for Array, but we should have a library that does that.
There are several other cases like this in our code, and we don't implement them in the right way just cause we don't have the algorithms at hand.

Reply:
yes, good one, maybe in another PR and if we have more cases like this, then applieing to all